parser/src/cfg/mod.rs

```diff
@@ -2948,34 +2943,29 @@ fn parse_layers(
                 }
             }
         }
-        for (layer_action, defsrc_action) in
-            layers_cfg[layer_level * 2][0].iter_mut().zip(defsrc_layer)
-        {
-            // Set transparent actions in the "layer-switch" version of the layer according to
-            // defsrc action.
-            if *layer_action == Action::Trans {
-                if s.block_unmapped_keys {
-                    *layer_action = Action::NoOp;
-                } else {
-                    *layer_action = defsrc_action;
-                }
+        for layer_action in layers_cfg[layer_level][0].iter_mut() {
+            if *layer_action == Action::Trans && s.block_unmapped_keys {
+                *layer_action = Action::NoOp;
             }
         }
         // Set fake keys on the `layer-switch` version of each layer.
```

Comment:
this comment is outdated

Reply:
https://github.com/jtroo/kanata/pull/938